### DISCUSSION TITLE
Can multiple date formats be used in meta data?

### DISCUSSION THREAD

**Discussion by Wysardry** [estruyf/vscode-front-matter#592] [Q&A]
I would like to have both machine readable and human readable date fields in the meta data. For example, "2023-06-24" in one field and "24th June, 2023" in another.

Is this possible? The docs say the date format can be set, but it isn't clear whether that is a global or per field setting.

Also, is Front Matter available for the Raspberry Pi?

**Comment by estruyf** [comment]
Hi @Wysardry, we don't have this kind of functionality in the main version yet, but earlier this week, it got implemented in the BETA version. Check it out here: https://github.com/estruyf/vscode-front-matter/issues/591

Feel free to try it and let me know how it works.

> Also, is Front Matter available for the Raspberry Pi?

If VSCode can run on it, I don't see a problem.

**Comment by Wysardry** [comment]
Thanks. That seems to be what I was looking for.

Is there currently a way to link two fields together so that if you modify one, the other updates automatically? This is also to do with datetime.

For example, I would like to be able to click the "now" button on a "modified-datetime" field and have both the "modified-datetime" and the "modified-date" field data get updated.

It's no big deal if it isn't possible. In my use case, I would just have to avoid changing the fields near midnight.

**Reply by estruyf** [reply]
No, there is no field or setting that would support this use case at the moment. If there are others who need this kind of functionality or if there is another use case, we can put it on the backlog.

**Comment by Wysardry** [comment]
I can't think of another use case off the top of my head.

The problem I'm trying to solve is that the date formatting options for many static site generators does not include adding a suffix (st, nd, rd or th) to the day of the month.

Adding new fields and editing the dates manually solves this, but that naturally means editing twice as many fields. As my site will be a digital garden rather than a blog, articles will be updated on a regular basis.

It's quite a specific use case (digital garden creators who are fussy about date formatting), but it does apply to several SSGs.

Even without the field linking feature, Front Matter will definitely make updating sites easier, so thank you for your efforts so far.
